app/controllers/concerns/api_token_authenticatable.rb

```diff
@@ -50,6 +50,6 @@ def check_access_scope
   end
 
   def access_scope
```

Comment:
if we have a look at controllers like `DataStudio::SchoolRolloutController` these have:
```
def access_scope
  ApiToken.where(private_api_access: true)
end
```
Which we want to keep. Can we keep this as is and keep the level the same everywhere and just remove the NPQ access please instead?